diff --git a/node_modules/@react-native-async-storage/async-storage/android/build.gradle b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
index 4560dc7..601e5b3 100644
--- a/node_modules/@react-native-async-storage/async-storage/android/build.gradle
+++ b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
@@ -4,7 +4,7 @@ configurations {
 
 buildscript {
     apply from: "config.gradle"
-    def kotlinVersion = ext.AsyncStorageConfig.kotlinVersion
+    def kotlinVersion = "1.8.22"
     def kspVersion = ext.AsyncStorageConfig.kspVersion
 
     repositories {
